src/main/java/com/blackwaterpragmatic/webservices/resource/AbstractResourceResponseBuilder.java

```diff
@@ -0,0 +1,40 @@
+package com.blackwaterpragmatic.webservices.resource;
+
+
+import com.blackwaterpragmatic.webservices.bean.Error;
+
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+public abstract class AbstractResourceResponseBuilder<T> {
```

Comment:
I thought it might make sense to encapsulate the Response building boiler plate to a separate class, with multiple implementation (i.e. one per Resource)